pkg/jobrunaggregator/jobrunaggregatoranalyzer/cmd.go

```diff
@@ -51,6 +52,7 @@ func (f *JobRunsAnalyzerFlags) BindFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&f.ExplicitGCSPrefix, "explicit-gcs-prefix", f.ExplicitGCSPrefix, "only used by per PR payload promotion jobs.  This overrides the well-known mapping and becomes the required prefix for the GCS query")
 	fs.DurationVar(&f.Timeout, "timeout", f.Timeout, "Time to wait for aggregation to complete.")
 	fs.StringVar(&f.EstimatedJobStartTimeString, "job-start-time", f.EstimatedJobStartTimeString, fmt.Sprintf("Start time in RFC822Z: %s", kubeTimeSerializationLayout))
+	fs.StringVar(&f.AggrDataTime, "aggr-data-start-time", f.AggrDataTime, fmt.Sprintf("Aggregation data start time (used to target specific aggregation data) in RFC822Z: %s", kubeTimeSerializationLayout))
```

Comment:
Is there any reason not to use RFC3339     = "2006-01-02T15:04:05Z07:00", this seems much more common and intuitive to me. I was not immediately aware what this format was.

Reply:
apparently, we're already using RFC3339 but not saying it in the text.  Line 41 shows `const kubeTimeSerializationLayout = time.RFC3339`.  So I fixed the text to match.